build/calculateContainerSizes.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
This script probably doesn't need to exist in the repository, but I made it in order to pull data about how large the containers are after installing all of the build tools during the build script, so we can compare to baking everything in.
Results for cli image:
```
Sample app container sizes for oryxdevmcr.azurecr.io/public/oryx/cli
-------------------------------------------
DotNetCore/NetCoreApp31.MvcApp size: 868MB (virtual 1.17GB)
golang/hello-world size: 480MB (virtual 776MB)
hugo/hugo-sample size: 53.6MB (virtual 350MB)
java/MavenSimpleJavaApp size: 443MB (virtual 740MB)
nodejs/helloworld-nuxtjs size: 681MB (virtual 977MB)
php/greetings size: 290MB (virtual 587MB)
python/flask-app size: 518MB (virtual 815MB)
ruby/Jekyll-app size: 612MB (virtual 909MB)
```

Reply:
```
Sample app container sizes for oryxdevmcr.azurecr.io/public/oryx/cli-buster
-------------------------------------------
DotNetCore/NetCoreApp31.MvcApp size: 925MB (virtual 1.24GB)
golang/hello-world size: 509MB (virtual 827MB)
hugo/hugo-sample size: 53.6MB (virtual 371MB)
java/MavenSimpleJavaApp size: 473MB (virtual 790MB)
nodejs/helloworld-nuxtjs size: 737MB (virtual 1.05GB)
php/greetings size: 254MB (virtual 571MB)
python/flask-app size: 565MB (virtual 882MB)
ruby/Jekyll-app size: 686MB (virtual 1GB)
```